src/diffusers/loaders.py

```diff
@@ -1540,9 +1540,11 @@ def _convert_kohya_lora_to_diffusers(cls, state_dict):
                 diffusers_name = diffusers_name.replace("proj.in", "proj_in")
                 diffusers_name = diffusers_name.replace("proj.out", "proj_out")
                 diffusers_name = diffusers_name.replace("emb.layers", "time_emb_proj")
+                diffusers_name = diffusers_name.replace("time.emb.proj", "time_emb_proj")
+                diffusers_name = diffusers_name.replace("conv.shortcut", "conv_shortcut")
 
                 # SDXL specificity.
-                if "emb" in diffusers_name:
+                if "emb" in diffusers_name and "time_emb_proj" not in diffusers_name:
```

Comment:
Could you please justify this change?

Reply:
There are time_emb_proj (https://huggingface.co/CompVis/stable-diffusion-v1-4/blame/3b6228db95406c454e8642f785588ba45a6057b4/unet/down_blocks.0.resnets.0.time_emb_proj.weight) weight in resnet block. For those models, the logic inside if makes weight unloadable. This logic should be only applied to SDXL